azext_iot/_help.py

```diff
@@ -576,6 +576,49 @@
         -m {module_name} -j ../mymodtwin.json
 """
 
+helps[
+    "iot hub state"
+] = """
+    type: group
+    short-summary: Download, upload or migrate the state of an IoT Hub.
+"""
+
+helps[
+    "iot hub state export"
+] = """
+    type: command
+    short-summary: Saves the configuration and device information of an IoT Hub to a file.
```

Comment:
Save

Reply:
also say device information and device twin - they are technically different terminology

Reply:
Just curious any issue with using 'Export(s) ...' in the description matching the command verb? Export is a fine word.

In the short-summary saying something like

"Exports the state of an IoT Hub to a file."

Then optionally the long summary could spell out what the state entails.

"The exported state will include device and module identities as well as their twin tags and desired properties, ADM configurations, edge deployments, .... ' etc etc.

Reply:
Agree with paymaun - make the short summary just say that it will save the state and the long summary detail what the state is